game.py

```diff
@@ -20,7 +21,31 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+
+    wrong_guesses_list = []
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+
+    while len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES:
+        user_input = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
+
+        if user_input in snowman_word:
+            print("You guessed a letter that's in the word!")
+            #log correct letter guess
+            correct_letter_guess_statuses[user_input] = True
+            #check if complete
+            if is_word_guessed(snowman_word, correct_letter_guess_statuses):
+                print(f"Congratulations, you win!")
+                return
+        else:
+            print(f"The letter {user_input} is not in the word")
+            #instead of counting wrong guesses, add the guess to wrong_guesses_list
+            wrong_guesses_list.append(user_input)
```

Comment:
```suggestion
            wrong_guesses_list.append(user_input)
            print(f"The letter {user_input} is not in the word")
            # instead of counting wrong guesses, add the guess to wrong_guesses_list
```